src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -61,6 +61,7 @@
 
     private static boolean standaloneEnabled = true;
     private static boolean reconfigEnabled = false;
+    private static int requestWarnThresholdMs = 10000;
```

Comment:
If we're going to implement this let's at least put some sort of realistic threshold. 10s is basically saying "don't enable this feature" is that what we want?